src/dmtcp_restart.cpp

```diff
@@ -801,7 +801,6 @@ int main(int argc, char** argv)
   bool foundNonOrphan = false;
   RestoreTargetMap::iterator it;
   int size = independentProcessTreeRoots.size();
```

Comment:
Let's remove this as well. I don't think this `size` variable is being used anywhere else.